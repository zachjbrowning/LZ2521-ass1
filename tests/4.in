s a 
s ajouter
s aj
l