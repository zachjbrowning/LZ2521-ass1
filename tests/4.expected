> Usage: s <string>
> ajouter to add
> ajouter to add
> 2 ajouter to add
> 